operator/handler/config.go

```diff
@@ -45,7 +45,24 @@ type sidecarConfig struct {
 }
 
 // getConfigs gets configurations from TailingSidecars and annotations
-func getConfigs(annotations map[string]string, tailingSidecars []tailingsidecarv1.TailingSidecarConfig) []sidecarConfig {
+func getConfigs(annotations map[string]string, tailingSidecarConfigs []tailingsidecarv1.TailingSidecarConfig) ([]sidecarConfig, error) {
+	configs := parseAnnotation(annotations)
```

Comment:
put that below `convertTailingSidecarConfigs` call, you might save some cycles when there's an error 🤷